app/core/config.py

```diff
@@ -49,9 +45,7 @@ def assemble_db_connection(cls, v: Optional[str], values) -> Any:
         .has().no().spaces() \
 
     ACCESS_TOKEN_EXPIRATION_TIME: int
-    KEY: str = "c2bab29d257f0ffc52d9ac677d4ff6d1d9d5e92e3d3939d3f4cwc"
-
-    # model_config = SettingsConfigDict(case_sensitive=True, env_file="defaults.env")
+    JWT_KEY: str = "c2bab29d257f0ffc52d9ac677d4ff6d1d9d5e92e3d3939d3f4cwc"
```

Comment:
Hmmm maybe this is a good idea to add it in the `.env` file, since when using the gatekeeper the JWT_KEY should be the same one used in the gatekeeper, so that the service can decode the JWTtoken sent by the gatekeeper.